libs/feature-flags/src/lib/features.ts

```diff
@@ -48,6 +48,9 @@ export enum Features {
 
   // Search indexer
   shouldSearchIndexerResolveNestedEntries = 'shouldSearchIndexerResolveNestedEntries',
+
+  // Userprofile Collection
+  isIASSpaPagesEnabled = 'isiasspapagesenabled',
```

Comment:
This is one line lower then in the hide modal PR so this will probably conflict when that PR is merged to main.